🐛 Bug Report: Incorrect Java Syntax

### 👟 Reproduction steps

Missing `public class Index {...}`

### 👍 Expected behavior

`public class Index {...}`

### 👎 Actual Behavior

Missing `public class Index {...}`

### 🎲 Appwrite version

Version 1.0.x

### 💻 Operating system

Linux

### 🧱 Your Environment

_No response_

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

I got it, Open Runtimes will run `build.sh` script which will generate Wrapper.java class from Index.java template.

However, template (in this repository) contains bug, `env` is not defined. 
```
Docker Error: /usr/local/src/src/main/java/io/openruntimes/java/Wrapper.java:53: error: cannot find symbol
    if (env == null
        ^
  symbol:   variable env
  location: class Wrapper
```

So that instead of this one, I use https://github.com/open-runtimes/open-runtimes/blob/main/runtimes/java-11.0/example/Index.java 

Also, RuntimeRequest & RuntimeResponse are here:
https://github.com/open-runtimes/open-runtimes/tree/main/runtimes/java-11.0/src/main/java/io/openruntimes/java

[stnguyen90](https://github.com/stnguyen90),
`env` is undefined. It is HashMap? TreeMap? what is `env`?
https://github.com/open-runtimes/open-runtimes/tree/main/runtimes/java-11.0

I follow step by step tutorial to generate function and deploy it, I didn't change any code in provided autogenerated example function and it cannot be built.

```
Docker Error: /usr/local/src/src/main/java/io/openruntimes/java/Wrapper.java:53: error: cannot find symbol
    if (env == null
        ^
  symbol:   variable env
  location: class Wrapper
/usr/local/src/src/main/java/io/openruntimes/java/Wrapper.java:54: error: cannot find symbol
        || !env.containsKey("APPWRITE_FUNCTION_ENDPOINT")
            ^
  symbol:   variable env
  location: class Wrapper
/usr/local/src/src/main/java/io/openruntimes/java/Wrapper.java:55: error: cannot find symbol
        || !env.containsKey("APPWRITE_FUNCTION_API_KEY")
            ^
  symbol:   variable env
  location: class Wrapper
Note: /usr/local/src/src/main/java/io/openruntimes/java/RuntimeRequest.java uses unchecked or unsafe operations.
Note: Recompile with -Xlint:unchecked for details.
 errors

FAILURE: Build failed with an exception.
```

--- a/java-11.0/src/Index.java
+++ b/java-11.0/src/Index.java
@@ -18,13 +18,13 @@ import io.appwrite.services.Users;
 
 /*
   'req' variable has:
-    'headers' - object with request headers
-    'payload' - request body data as a string
-    'variables' - object with function variables
+    'getHeaders()' - function to get headers as a Map<String, String>
+    'getPayload()' - function to get body data as a String
+    'getVariables()' - function to get variables as a Map<String, String>
 
   'res' variable has:
-    'send(text, status)' - function to return text response. Status code defaults to 200
-    'json(obj, status)' - function to return JSON response. Status code defaults to 200
+    'send(text, status)' - function that accepts a String to return text response. Status code defaults to 200
+    'json(obj, status)' - function that accepts a Map<String, Object> to return JSON response. Status code defaults to 200
   
   If an error is thrown, a response with code 500 will be returned.
 */
@@ -47,9 +47,9 @@ public RuntimeResponse main(RuntimeRequest req, RuntimeResponse res) throws Exce
 
     var variables = req.getVariables();
 
-    if (env == null
-        || !env.containsKey("APPWRITE_FUNCTION_ENDPOINT")
-        || !env.containsKey("APPWRITE_FUNCTION_API_KEY")
+    if (variables == null
+        || !variables.containsKey("APPWRITE_FUNCTION_ENDPOINT")
+        || !variables.containsKey("APPWRITE_FUNCTION_API_KEY")
         || variables.get("APPWRITE_FUNCTION_ENDPOINT") == null
         || variables.get("APPWRITE_FUNCTION_API_KEY") == null) {
         System.out.println("Environment variables are not set. Function cannot use Appwrite SDK.");

--- a/java-17.0/src/Index.java
+++ b/java-17.0/src/Index.java
@@ -18,13 +18,13 @@ import io.appwrite.services.Users;
 
 /*
   'req' variable has:
-    'headers' - object with request headers
-    'payload' - request body data as a string
-    'variables' - object with function variables
+    'getHeaders()' - function to get headers as a Map<String, String>
+    'getPayload()' - function to get body data as a String
+    'getVariables()' - function to get variables as a Map<String, String>
 
   'res' variable has:
-    'send(text, status)' - function to return text response. Status code defaults to 200
-    'json(obj, status)' - function to return JSON response. Status code defaults to 200
+    'send(text, status)' - function that accepts a String to return text response. Status code defaults to 200
+    'json(obj, status)' - function that accepts a Map<String, Object> to return JSON response. Status code defaults to 200
   
   If an error is thrown, a response with code 500 will be returned.
 */
@@ -47,9 +47,9 @@ public RuntimeResponse main(RuntimeRequest req, RuntimeResponse res) throws Exce
 
     var variables = req.getVariables();
 
-    if (env == null
-        || !env.containsKey("APPWRITE_FUNCTION_ENDPOINT")
-        || !env.containsKey("APPWRITE_FUNCTION_API_KEY")
+    if (variables == null
+        || !variables.containsKey("APPWRITE_FUNCTION_ENDPOINT")
+        || !variables.containsKey("APPWRITE_FUNCTION_API_KEY")
         || variables.get("APPWRITE_FUNCTION_ENDPOINT") == null
         || variables.get("APPWRITE_FUNCTION_API_KEY") == null) {
         System.out.println("Environment variables are not set. Function cannot use Appwrite SDK.");

--- a/java-18.0/src/Index.java
+++ b/java-18.0/src/Index.java
@@ -18,13 +18,13 @@ import io.appwrite.services.Users;
 
 /*
   'req' variable has:
-    'headers' - object with request headers
-    'payload' - request body data as a string
-    'variables' - object with function variables
+    'getHeaders()' - function to get headers as a Map<String, String>
+    'getPayload()' - function to get body data as a String
+    'getVariables()' - function to get variables as a Map<String, String>
 
   'res' variable has:
-    'send(text, status)' - function to return text response. Status code defaults to 200
-    'json(obj, status)' - function to return JSON response. Status code defaults to 200
+    'send(text, status)' - function that accepts a String to return text response. Status code defaults to 200
+    'json(obj, status)' - function that accepts a Map<String, Object> to return JSON response. Status code defaults to 200
   
   If an error is thrown, a response with code 500 will be returned.
 */
@@ -47,9 +47,9 @@ public RuntimeResponse main(RuntimeRequest req, RuntimeResponse res) throws Exce
 
     var variables = req.getVariables();
 
-    if (env == null
-        || !env.containsKey("APPWRITE_FUNCTION_ENDPOINT")
-        || !env.containsKey("APPWRITE_FUNCTION_API_KEY")
+    if (variables == null
+        || !variables.containsKey("APPWRITE_FUNCTION_ENDPOINT")
+        || !variables.containsKey("APPWRITE_FUNCTION_API_KEY")
         || variables.get("APPWRITE_FUNCTION_ENDPOINT") == null
         || variables.get("APPWRITE_FUNCTION_API_KEY") == null) {
         System.out.println("Environment variables are not set. Function cannot use Appwrite SDK.");

--- a/java-8.0/src/Index.java
+++ b/java-8.0/src/Index.java
@@ -18,13 +18,13 @@ import io.appwrite.services.Users;
 
 /*
   'req' variable has:
-    'headers' - object with request headers
-    'payload' - request body data as a string
-    'variables' - object with function variables
+    'getHeaders()' - function to get headers as a Map<String, String>
+    'getPayload()' - function to get body data as a String
+    'getVariables()' - function to get variables as a Map<String, String>
 
   'res' variable has:
-    'send(text, status)' - function to return text response. Status code defaults to 200
-    'json(obj, status)' - function to return JSON response. Status code defaults to 200
+    'send(text, status)' - function that accepts a String to return text response. Status code defaults to 200
+    'json(obj, status)' - function that accepts a Map<String, Object> to return JSON response. Status code defaults to 200
   
   If an error is thrown, a response with code 500 will be returned.
 */
@@ -47,9 +47,9 @@ public RuntimeResponse main(RuntimeRequest req, RuntimeResponse res) throws Exce
 
     Map<String, String> variables = req.getVariables();
 
-    if (env == null
-        || !env.containsKey("APPWRITE_FUNCTION_ENDPOINT")
-        || !env.containsKey("APPWRITE_FUNCTION_API_KEY")
+    if (variables == null
+        || !variables.containsKey("APPWRITE_FUNCTION_ENDPOINT")
+        || !variables.containsKey("APPWRITE_FUNCTION_API_KEY")
         || variables.get("APPWRITE_FUNCTION_ENDPOINT") == null
         || variables.get("APPWRITE_FUNCTION_API_KEY") == null) {
         System.out.println("Environment variables are not set. Function cannot use Appwrite SDK.");

